apps/web/app/admin/layout.tsx

```diff
@@ -0,0 +1,7 @@
+interface AdminLayout {
+  children: React.ReactNode;
+}
```

Comment:
```suggestion
interface AdminLayout {
  children: ReactNode;
}
```
프론트랑 초반에 이야기했던 부분인데 import해서 사용하기로했어!
이렇게 작성하는게 우리 컨벤션에 맞는 부분일거같아